### PR TITLE
CLIMATE-861 - An example to plot aerosol optical depth climatology from NASA MISR instrument

### PR DESCRIPTION
**Pull request by huikyole**
- A new example, examples/draw_climatology_map_MISR_AOD.py, has been added.

### CODE REVIEW

File: examples/draw_climatology_map_MISR_AOD.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import ocw.dataset as ds
```
Comment (lewismc) [inline examples/draw_climatology_map_MISR_AOD.py]: License header please

File: examples/draw_climatology_map_MISR_AOD.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import ocw.data_source.local as local
+import ocw.dataset_processor as dsp
+import ocw.plotter as plotter
+
+import numpy as np
+import numpy.ma as ma
+
+
+''' data source: https://dx.doi.org/10.6084/m9.figshare.3753321.v1'''
+dataset = local.load_file('/home/huikyole/climate/examples/AOD_monthly_2000-MAR_2016-FEB_from_MISR_L3_JOINT.nc',
```
Comment (lewismc) [inline examples/draw_climatology_map_MISR_AOD.py]: This relative path needs to be replaced ideally by a remote, publicly available one @huikyole

File: examples/draw_climatology_map_MISR_AOD.py (hunk 1 above)
Comment (huikyole) [inline examples/draw_climatology_map_MISR_AOD.py]: Thank you for the reminder. I will add the license header.

File: examples/draw_climatology_map_MISR_AOD.py (hunk 2 above)
Comment (huikyole) [inline examples/draw_climatology_map_MISR_AOD.py]: I will update it. Users can download the data from the doi in the above though. Thx.

**Pull request merged**